OptimoveSDK/Sources/Classes/Optimobile/Optimobile.swift

```diff
@@ -116,6 +119,28 @@ class Optimobile {
         maybeAlignUserAssociation(initialUserId: initialUserId)
     }
 
+    static func completeDelayedConfiguration(config: OptimobileConfig) {
+        guard let credentials = config.credentials else {
+            Logger.warn("No credentials provided for delayed configuration completion.")
+            return
+        }
+        Logger.info("Completing delayed configuration with credentials: \(credentials)")
+        updateStorageValues(config)
+        setCredentials(credentials)
+        NotificationCenter.default.post(name: .optimobileInializationFinished, object: nil)
```

Comment:
another thing android does is syncing in-apps

Reply:
Added `sync` invocation in the in-app for `.optimobileInializationFinished` notification.